android-sdk/src/test/java/org/sagebionetworks/bridge/android/manager/auth/InMemoryDAO.java

```diff
@@ -15,7 +18,7 @@
  * Created by jyliu on 1/20/2017.
  */
 @AnyThread
-class InMemoryDAO implements AuthenticationManager.DAO {
+class InMemoryDAO implements AccountDAO {
```

Comment:
This is used by unit tests to avoid mocking get/set repeatedly or using RobolectricRunner to have LocalStorage. Is there a better approach? It's been several years since I looked